src/main/java/PDLayer.mdl

```diff
@@ -869,6 +869,8 @@ type PDStructElem extends PDStructTreeNode {
 	property parentStandardType: String;
 	% structure type
 	link S: CosUnicodeName;
+	% the namespace and the tag within a given namespace separated by colon
```

Comment:
Which exactly namespace is meant here? Does this include the default (PDF 1.7) namespace if no namespaces explicitly specified?

Reply:
Yes, this include the default namespace

Reply:
I'd suggest this description: "the namespace specified in the element dictionary (or the default namespace, if not explicitly specified) and the element tag separated by the colon"